pkg/crc/constants/constants_darwin.go

```diff
@@ -12,7 +12,14 @@ const (
 )
 
 var (
-	TrayExecutablePath   = filepath.Clean(filepath.Join(version.InstallPath(), "..", "MacOS", "CodeReady Containers"))
 	TapSocketPath        = filepath.Join(CrcBaseDir, "tap.sock")
 	DaemonHTTPSocketPath = filepath.Join(CrcBaseDir, "crc-http.sock")
 )
+
+func TrayExecutablePath() string {
+	if version.IsInstaller() {
+		return filepath.Clean(filepath.Join(version.InstallPath(), "..", "MacOS", "CodeReady Containers"))
+	}
+	// Should not be reached, tray is only supported on installer builds
+	return filepath.Clean(filepath.Join(BinDir(), "CodeReady Containers"))
+}
```

Comment:
@cfergeau As per commit message `The code is calling InstallPath() even on crc versions where it was not
set at build time. Since InstallPath() is printing a warning when this occurs, it's better if we can avoid this.`  Does it mean that when this PR is in this can still occur because we are already setting it in Makefile right?

Reply:
If you build crc from git on macos, `TrayExecutablePath` was initialized using `InstallPath()` unconditionally even if the tray code can only be triggered in "install" builds. So a warning was printed each time `crc` was started. By moving it to a function, this will only be called when needed, which should be never on git builds, so it's only called when building an installer.